services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/controller/MacControllerTest.java

```diff
@@ -108,7 +111,8 @@ public String createMacRange(MacRange macRange) {
     public void test_index() throws Exception {
         this.mockMvc.perform(get("/start.html"))
                 .andDo(print())
-                .andExpect(status().isOk());
+                .andExpect(status().isOk())
+                .andDo(document("start"));
```

Comment:
should we use "mac_range_post"?

Reply:
* This corresponds to the directory name of the generated document. I think the name and the request are the same for easy search. I can change it if necessary, but it cannot conflict with other requests, otherwise it will overwrite the file